modules/gmc-agent/src/main/java/org/gridgain/action/query/QueryHolder.java

```diff
@@ -29,6 +29,9 @@
     /** Query ID. */
     private final String qryId;
 
+    /** Is scan query. */
+    private final boolean isScanQry;
```

Comment:
I think we can store this in CursorHolder and don't work with QueryHolder

Reply:
Move.